service/vspo-schedule/web/src/components/Head/Head.tsx

```diff
@@ -23,7 +23,7 @@ export const CustomHead = ({
       <meta property="og:description" content={description} />
       <meta
         property="og:url"
-        content={`https://www.vspo-schedule.com/${path}`}
+        content={`https://www.vspo-schedule.com${router.asPath}`}
```

Comment:
Maybe we could use related path here?
```diff
- content={`https://www.vspo-schedule.com${router.asPath}`}
+ content={`/${router.asPath}`}
```
Or nextjs's environment varaiable so that we won't need to modify this line if we want to change domain